### ISSUE TITLE
No option to select which AMS slot to use to print from.

### ISSUE THREAD

**Issue by CallmeBigfat** [bigtreetech/PandaTouch#73]
Would love to see this option since changing plastic requires walking to the machine regardless. When I want to make the same object but in a different color.

Hopefully this is possible.

![image](https://github.com/bigtreetech/PandaTouch/assets/72172156/5aa2a63b-4726-4dfd-a758-4295cb61da4e)

**Comment by StefanX13** [comment]
+1

**Comment by insane78** [comment]
Yes, like X1C we should have the possibility to remap the colors of the slice to those present in ams 
![image](https://github.com/bigtreetech/PandaTouch/assets/49029448/cacfa319-a4cc-4ffa-a586-7c2b32b7cca7)

**Comment by mike391** [comment]
In the past Panda Touch devs said that they could not do this without reslicing, however it is possible to define `ams_mapping` in the print request:

https://github.com/Doridian/OpenBambuAPI/blob/main/mqtt.md?plain=1

![Image](https://github.com/user-attachments/assets/15e5da94-7155-470a-a276-cd18b45edd87)

**Comment by dilixi** [comment]
> Would love to see this option since changing plastic requires walking to the machine regardless. When I want to make the same object but in a different color.
> 
> Hopefully this is possible.

![Image](https://github.com/user-attachments/assets/a4b3da4c-c61e-44af-8c48-1ce98cf269c6)

Please have a try with this one.

https://github.com/bigtreetech-beta/PandaTouch-Support/blob/main/firmware/v1.0.7.1_beta6.bin

It will supports multi-AMS later.
The printing task needs to be started by bambu studio at the first time.

**Comment by mike391** [comment]
Thanks! What .img do we use?

**Comment by dilixi** [comment]
> Thanks! What .img do we use?
Hi,
You can use this one:
https://github.com/bigtreetech/PandaTouch/blob/master/Firmware/1.0.7/en.img

> Thanks! What .img do we use?

Hi,
You can use this one:
https://github.com/bigtreetech/PandaTouch/blob/master/Firmware/1.0.7/en.img
